src/helpers/api.ts

```diff
@@ -1,6 +1,7 @@
-import axios from 'axios';
+import axios, { AxiosResponse } from 'axios';
+import { AxiosOptions, Url } from './helpersInterfaces';
 
-export const URL = {
+export const URL: Url = {
```

Comment:
`Url` interface (and variable) name is a bit confusing, please rename it to define that it's a config for api routes